### PR TITLE
FTX: Fix order placement/cancellation issues

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

This PR includes bug fixes for FTX order placement/cancellation.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run
- [x] Manual order placement / cancellation

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Travis with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shazbert — APPROVED** [review]
Tested order submission: 
image
Cancelled by OrderID 
Cancelled by CustomOrderID

Approved 👍

File: exchanges/ftx/ftx_test.go [hunk 1]
```diff
@@ -356,6 +383,27 @@ func TestTriggerOrder(t *testing.T) {
 	}
 }
 
+func TestCancelOrder(t *testing.T) {
```
Comment (MadCozBadd) [inline exchanges/ftx/ftx_test.go:386]: can CancelOrder account for using ClientOrderID too? during my implementation i didnt add it, but can we have an if statement to see if ClientOrderID != ""  n then use DeleteOrderByClientID

**Review by MadCozBadd — COMMENTED** [review]
no issues as such, just couldnt use clientorderid to cancel an order using wrapper coz i forgot/didnt implement in my original pr

File: exchanges/ftx/ftx_test.go (hunk 1 above)
Comment (thrasher-) [reply exchanges/ftx/ftx_test.go:386]: Yep!

**Review by gloriousCode — APPROVED** [review]
tACK! Did some real tests and then cancelled those orders.
I'm sad that the submitorder test did not immediately get filled.

**Review by MadCozBadd — APPROVED** [review]
tACK

**Review by shazbert — APPROVED** [review]
New changes tACK - create via submit order and cancel via CancelOrder method.

**Pull request merged**